src/main/config/config.h

```diff
@@ -36,9 +36,10 @@ typedef enum {
 typedef struct pilotConfig_s {
     char name[MAX_NAME_LENGTH + 1];
     char displayName[MAX_NAME_LENGTH + 1];
+    uint8_t modelId;
 } pilotConfig_t;
 
-PG_DECLARE(pilotConfig_t, pilotConfig);
+PG_DECLARE(pilotConfig_t, pilotConfig);   
```

Comment:
Please don't add random white space in random places!!!

Reply:
sorry i'm the "editor" guessing and pecking without knowing the normal do's and don'ts. i hope this is it now. thanks for time